tests/performance/locust_settings/_locust_settings.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable=unused-argument
+# pylint: disable=no-self-use
+# pylint: disable=no-name-in-module
+
+from datetime import timedelta
+
+from parse import Result, parse
+from pydantic import (
+    AnyHttpUrl,
+    Field,
+    PositiveFloat,
+    PositiveInt,
+    SerializationInfo,
+    field_serializer,
+    field_validator,
+)
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+from ._dump_dotenv import dump_dotenv
+
+
+class LocustSettings(BaseSettings):
+    model_config = SettingsConfigDict(cli_parse_args=True)
+
+    LOCUST_HOST: AnyHttpUrl = Field(
+        default=..., examples=["https://api.osparc-master.speag.com"]
+    )
+    LOCUST_USERS: PositiveInt = Field(default=...)
```

Comment:
You mean number of users or user id?

Reply:
I cannot change the names of these because they are locust settings. But I can try to add an explanation here.